src/test/scala/com/johnsnowlabs/util/TrainingModelsScript.scala

```diff
@@ -0,0 +1,11 @@
+import com.johnsnowlabs.nlp.SparkAccessor
+import com.johnsnowlabs.pretrained.EnModelsTraining
+
+// ToDo 1. Downlaad datasets before if needs
```

Comment:
I tried training and zipping the models that do not require extra data. They seem to be working.
Maybe you can also put these datasets / aux data on S3.

Reply:
Alberto, I've uploaded models and datasets.